client/client.go

```diff
@@ -233,3 +245,41 @@ func (c *Client) loadShortID() error {
 	c.shortID = binary.LittleEndian.Uint32(data)
 	return nil
 }
+
+// readCTFile looks for a CT settings file, and if it is found,
+// parses it by line for the energy multiplier and divider.
+func (c *Client) readCTSettingsFile() error {
+	path := filepath.Join(c.staticBaseDir, CTSettingsFile)
+	file, err := os.Open(path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil // use defaults
+		} else {
+			return fmt.Errorf("error opening ct settings file: %v", err)
+		}
+	}
```

Comment:
In golang, we try to avoid nesting as much as possible. A more idiomatic to do this control flow would be:

```
file, err := os.Open(path)
if os.IsNotExist(err) {
    return nil
}
if err != nil {
    return fmt.Errorf("error opening ct settings file: %v", err)
}
defer file.Close()
```

Except, since we want to move the default setting logic into this file, it'd actually be:

```
file, err := os.Open(path)
if os.IsNotExist(err) {
    c.energyMultiplier = EnergyMultiplierDefault
    c.energyDivider = EnergyDividerDefault
    return nil
}
if err != nil {
    return fmt.Errorf("error opening ct settings file: %v", err)
}
defer file.Close()
```